### PR TITLE
Jcf collection, list / it-academy / JC1-110-22 / Evgeniy Antonovich / Done tasks collections and List

### PR DESCRIPTION
**Pull request by aevgeni**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [x] fixed
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/java/core/practice/jcf_collection/StringCollectionUtil.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    }
+
+    public void removeWordsByLength(Collection<String> strings, int wordSize) {
+        Iterator<String> iterator = strings.iterator();
+        while (iterator.hasNext()) {
+           if (iterator.next().length() == wordSize) {
+               iterator.remove();
+           }
+        }
+    }
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/jcf_collection/StringCollectionUtil.java:23]: fix

File: src/main/java/com/rakovets/course/java/core/practice/jcf_list/class_/Person.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Override
+    public String toString() {
+        return "Student{" +
+                "name='" + name + " " + surname + '\'' +
+                ", averageAnnualMark=" + averageAnnualMark +
+                ", age=" + age +
+                '}';
+    }
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/jcf_list/class_/Person.java:23]: move to bottom

File: src/main/java/com/rakovets/course/java/core/practice/jcf_list/studio/Actor.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Override
+    public String toString() {
+        return "Actor{" +
+                "name='" + firstName + " " + lastName + '\'' +
+                ", cash=" + fee +
+                ", age=" + age +
+                '}';
+    }
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/jcf_list/studio/Actor.java:23]: move to bottom

File: src/test/java/com/rakovets/course/java/core/practice/jcf_collection/StringCollectionUtilDemo.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package com.rakovets.course.java.core.practice.jcf_collection;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StringCollectionUtilDemo {
```
Comment (rakovets) [inline src/test/java/com/rakovets/course/java/core/practice/jcf_collection/StringCollectionUtilDemo.java:6]: Best way: tests

File: src/test/java/com/rakovets/course/java/core/practice/jcf_list/class_/ClassDemo.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
+package com.rakovets.course.java.core.practice.jcf_list.class_;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ClassDemo {
```
Comment (rakovets) [inline src/test/java/com/rakovets/course/java/core/practice/jcf_list/class_/ClassDemo.java:6]: Best way: tests

File: src/test/java/com/rakovets/course/java/core/practice/jcf_list/studio/StudioDemo.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
+package com.rakovets.course.java.core.practice.jcf_list.studio;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StudioDemo {
```
Comment (rakovets) [inline src/test/java/com/rakovets/course/java/core/practice/jcf_list/studio/StudioDemo.java:6]: Best way: tests

**Review by rakovets — CHANGES_REQUESTED** [review]

File: src/main/java/com/rakovets/course/java/core/practice/jcf_collection/StringCollectionUtil.java (hunk 1 above)
Comment (aevgeni) [reply src/main/java/com/rakovets/course/java/core/practice/jcf_collection/StringCollectionUtil.java]: fixed it

File: src/main/java/com/rakovets/course/java/core/practice/jcf_list/class_/Person.java (hunk 2 above)
Comment (aevgeni) [reply src/main/java/com/rakovets/course/java/core/practice/jcf_list/class_/Person.java]: did it

File: src/main/java/com/rakovets/course/java/core/practice/jcf_list/studio/Actor.java (hunk 3 above)
Comment (aevgeni) [reply src/main/java/com/rakovets/course/java/core/practice/jcf_list/studio/Actor.java]: did it

File: src/test/java/com/rakovets/course/java/core/practice/jcf_list/class_/ClassDemo.java (hunk 5 above)
Comment (aevgeni) [reply src/test/java/com/rakovets/course/java/core/practice/jcf_list/class_/ClassDemo.java:6]: got it

File: src/test/java/com/rakovets/course/java/core/practice/jcf_list/studio/StudioDemo.java (hunk 6 above)
Comment (aevgeni) [reply src/test/java/com/rakovets/course/java/core/practice/jcf_list/studio/StudioDemo.java:6]: got it

**Pull request closed without merging**